<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1375</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="RunButton">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>790</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>790</y>
      <width>1201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <widget class="QSlider" name="SliderThreshold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>351</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTextBrowser" name="ThresholdText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:704;&quot;&gt;Threshold&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBarThreshold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>351</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QTextBrowser" name="DensityText">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:704;&quot;&gt;Density&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBarDensity">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>351</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="format">
      <string>%v</string>
     </property>
    </widget>
    <widget class="QSlider" name="SliderDensity">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>80</y>
       <width>351</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="TryButton">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>TRY</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>151</width>
       <height>101</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="OffText">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:704;&quot;&gt;.off&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="PlyText">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:704;&quot;&gt;.ply&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="PCText">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:704;&quot;&gt;Point cloud&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkOffFiles">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkPC">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkPlyFiles">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <widget class="QMdiArea" name="for3DViewer">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>9</y>
      <width>1181</width>
      <height>761</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1375</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionAll_Classes"/>
     <addaction name="actionLIDAR_DATA"/>
    </widget>
    <addaction name="menuOpen"/>
   </widget>
   <widget class="QMenu" name="menuFolder">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuMethods">
     <property name="title">
      <string>Methods</string>
     </property>
     <addaction name="actionL2_Distance"/>
    </widget>
    <widget class="QMenu" name="menuView_Classes">
     <property name="title">
      <string>View Classes</string>
     </property>
     <addaction name="actionShow_Sampled_Points"/>
     <addaction name="actionShow_Bounding_Box"/>
     <addaction name="actionShow_Mesh"/>
    </widget>
    <addaction name="menuMethods"/>
    <addaction name="menuView_Classes"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuFolder"/>
  </widget>
  <action name="actionFiles2">
   <property name="text">
    <string>CaseStudy2</string>
   </property>
  </action>
  <action name="actionCaseStudy3">
   <property name="text">
    <string>CaseStudy3</string>
   </property>
  </action>
  <action name="actionCaseStudy4">
   <property name="text">
    <string>CaseStudy4</string>
   </property>
  </action>
  <action name="actionCaseStudy5">
   <property name="text">
    <string>CaseStudy5</string>
   </property>
  </action>
  <action name="actionCaseStudy6">
   <property name="text">
    <string>CaseStudy6</string>
   </property>
  </action>
  <action name="actionFile1">
   <property name="text">
    <string>File1</string>
   </property>
  </action>
  <action name="actionShow_Sampled_Points">
   <property name="text">
    <string>Show Sampled Points</string>
   </property>
  </action>
  <action name="actionShow_Bounding_Box">
   <property name="text">
    <string>Show Bounding Box</string>
   </property>
  </action>
  <action name="actionShow_Mesh">
   <property name="text">
    <string>Show Mesh</string>
   </property>
  </action>
  <action name="actionL2_Distance">
   <property name="text">
    <string>L2 Distance</string>
   </property>
  </action>
  <action name="actionCaseStudy2">
   <property name="text">
    <string>CaseStudy2</string>
   </property>
  </action>
  <action name="actionCaseStudy3_2">
   <property name="text">
    <string>CaseStudy3</string>
   </property>
  </action>
  <action name="actionCaseStudy4_2">
   <property name="text">
    <string>CaseStudy4</string>
   </property>
  </action>
  <action name="actionCaseStudy5_2">
   <property name="text">
    <string>CaseStudy5</string>
   </property>
  </action>
  <action name="actionCaseStudy6_2">
   <property name="text">
    <string>CaseStudy6</string>
   </property>
  </action>
  <action name="actionAll_Classes">
   <property name="text">
    <string>All Classes</string>
   </property>
  </action>
  <action name="actionLIDAR_DATA">
   <property name="text">
    <string>LIDAR DATA</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionAll_Classes</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_classes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBarThreshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>913</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>884</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderDensity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBarDensity</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>902</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>873</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TryButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>help_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>996</x>
     <y>855</y>
    </hint>
    <hint type="destinationlabel">
     <x>687</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLIDAR_DATA</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_lidar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>687</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RunButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>presence_code()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1295</x>
     <y>862</y>
    </hint>
    <hint type="destinationlabel">
     <x>687</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>cliquer(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>687</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>example_click_btn()</slot>
  <slot>example_click_btn2()</slot>
  <slot>help_button()</slot>
  <slot>open_classes()</slot>
  <slot>getsavefilename()</slot>
  <slot>open_lidar()</slot>
  <slot>presence_code()</slot>
  <slot>cliquer(QListWidgetItem*)</slot>
 </slots>
</ui>
